ray-curator/ray_curator/tasks/__init__.py

```diff
@@ -1,6 +1,14 @@
 from .document import DocumentBatch
 from .file_group import FileGroupTask
 from .image import ImageBatch, ImageObject
-from .tasks import EmptyTask, Task, _EmptyTask  # TODO: maybe this results in circular imports?
+from .tasks import EmptyTask, Task, _EmptyTask
 
-__all__ = ["DocumentBatch", "EmptyTask", "FileGroupTask", "ImageBatch", "ImageObject", "Task", "_EmptyTask"]
+__all__ = [
+    "DocumentBatch",
+    "EmptyTask",
+    "FileGroupTask",
+    "ImageBatch",
+    "ImageObject",
+    "Task",
+    "_EmptyTask",
```

Comment:
Why do we need to expose _EmptyTask directly?

Reply:
It was always there. `_EmptyTask` is the actual class so we need it for type hints while `EmptyTask` is the object. maybe the naming should be swapped